main.js

```diff
@@ -0,0 +1,66 @@
+/* Mobile Menu Interaction */
+const btnOpenMenu = document.getElementById('open-menu');
+const btnCloseMenu = document.getElementById('close-menu');
+const menu = document.getElementById('mobile-nav');
+btnOpenMenu.addEventListener('click', () => {
+  menu.style.left = '0';
+});
+btnCloseMenu.addEventListener('click', () => {
+  menu.style.left = '-100vw';
+});
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job implementing this transition from the left in your mobile menu I really liked it! 🥇 As a suggestion,  instead of adding those animations using `left`, `right`, `top`, or `bottom` properties, you could use `transform: translate`. You will get smoother and more performant animations.

If you're curious, here's an [article](https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108) about this topic!